tests/testthat/test-load_catch_dummy_kagoshima.R

```diff
@@ -0,0 +1,47 @@
+context("Load_catch_dummy_kagoshima")
+
+test_that("load_catch_dummy_kagoshima exits", {
+  expect_silent(load_catch_dummy_kagoshima())
+})
+
+test_that("load_catch_dummy_kagoshima is a data frame", {
+  expect_is(load_catch_dummy_kagoshima(), "data.frame")
+  expect_is(load_catch_dummy_kagoshima() %>%
+              dplyr::pull(Catch_ton), "integer")
+})
+
+context("- Numerical test")
+
+test_that("catch is created with one-by-one ascending number", {
+
+  prefec_catch_sum_test(
+    load_catch_dummy_kagoshima(),
+    prefec = "鹿児島", location = "阿久根", fishery = "まき網",
+    expected_data = 363378)
```

Comment:
たぶん、17行目で改行したこの場合には
```suggestion
    expected_data = 363378
  )
```
だと思います。細かいことなので、Suggest changeじゃなくてコメントです。